src/main/webapp/components/sources-info-local/sources-info-local.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+
+import { useQuery } from '@apollo/react-hooks'
+import gql from 'graphql-tag'
+
+import StorageIcon from '@material-ui/icons/Storage'
+import OnlineIcon from '@material-ui/icons/CloudDoneOutlined'
+import OfflineIcon from '@material-ui/icons/OfflineBoltOutlined'
+import { SourcesInfo } from '../sources-info/sources-info'
+
+const sources = gql`
+  query SourcesInfoLocal($attribute: String!) {
+    facet(attribute: $attribute) {
+      value
+      count
+    }
+    sources {
+      isAvailable
+      sourceId
+      local
+    }
+  }
+`
+
+const pollInterval = gql`
+  query SourcePollInterval {
```

Comment:
We ran into an issue a little while back from having gql tags with the same query names: https://github.com/connexta/revelio/pull/239

Can we pull this tag and the `useSourcePollInterval` hook into its own file and share it between `sources-info` and `sources-info-local`?